pkgs/development/tools/build-managers/drake-factual/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, leiningen, jdk, which, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "drake";
+  numVersion = "1.0.3";
+  version = "v${numVersion}";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "Factual";
+    rev = version;
+    sha256 = "0sm8ghw9x7ba05q6c2gx1my46xigw0a0djjdlrqni2jj3pf0v27w";
+  };
+
+  buildInputs = [ leiningen ];
+  propagatedBuildInputs = [ jdk which ];
```

Comment:
`propagatedBuildInputs` is not the right way of specifying run-time dependencies. Use makeWrapper as described in https://nixos.org/nixpkgs/manual/#sec-language-java